src/App/ConsolePage.xaml.cs

```diff
@@ -30,6 +30,11 @@ public ConsolePage()
             _outSem = new SemaphoreSlim(1, 1);
             _activeRunSem = new SemaphoreSlim(1, 1);
             _newCmd = false;
+            _cmdHistory = new List<string>(20);

```

Comment:
Can we initialize size of the list to MaxCmdHistory .

Reply:
my theory was 20 is the likely upper bound for most people so i init to that, but let you save up to 100 (i'd just need to resize the list).

In either way, i think its a very minor (insignificant) memory/perf trade off, and I'd rather err on the side of using less initial memory at the cost of the list resize perf hit instead of defaulting to use more memory.

tldr - likely irrelevent at this scale, but wont fixing to save a bit of memory.